Podfile

```diff
@@ -5,7 +5,7 @@ use_frameworks!
 
 target 'Tempura' do
   platform :ios, '9.0'
-  podspec
+  pod 'Katana', :git => 'https://github.com/BendingSpoons/katana-swift', :branch => 'experimental'
```

Comment:
I think you should wait for katana to be released and then update the podfile accordingly.

Reply:
Done!